WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -735,7 +737,8 @@ static GSourceFuncs _handlerIntervention =
             Core::JSON::DecUInt16 LocalStorageSize;
             Core::JSON::Boolean IndexedDBEnabled;
             Core::JSON::String IndexedDBPath;
-            Core::JSON::DecUInt16 IndexedDBSize; // [KB]
+            Core::JSON::DecUInt16 IndexedDBSize; // [percentage of volume space for each domain]
```

Comment:
Wouldn't a uint8 be enough for a percentage (0 to 100)?

Reply:
Ah wait I guess this is to keep the old existing config files that  still have this as 16 bit number readable right?

Reply:
This is why I initially left it was it used to be, so `DecUInt16` and not just uint8, but perhaps we could still change it to uint8, @modeveci? Depends whether we are positive that nobody was using it so far anyway

Reply:
uint8 is good. 
Beside that can we rename indexdbsize to originstorageration and indexdbtotalsize to totalstorageratio?